content/en/docs/collector/internal-telemetry.md

```diff
@@ -260,6 +260,57 @@ categorized by instrumentation type.
 | `rpc_server_response_size`        | Measures the size of RPC response messages (uncompressed).                                | Histogram |
 | `rpc_server_responses_per_rpc`    | Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.     | Histogram |
 
+### Telemetry maturity levels
+
+#### Tracing
+
+Tracing instrumentation is still under active development, and changes might be made to span names, attached attributes, instrumented endpoints, or other aspects of the telemetry. Until this feature graduates to stable, there are no guarantees of backwards compatibility for tracing instrumentation.
+
+#### Metrics
+
+The Collector's metrics follow a four-stage lifecycle:
+
+> Alpha metric → Stable metric → Deprecated metric → Deleted metric
```

Comment:
How does a user identify the stage in which a metric is at?

Reply:
This could be included in the documentation generated for each metric via the metadata file, wdyt?

Reply:
Yeah that makes sense. I would also like to have the stage in the metrics description

Reply:
Would you expect to see all the stages in the description or would it be enough to include it only in the deprecation stage? I wouldn't want to end up sending unnecessary prefixes for every metric at every stage.... maybe any stage that isn't stable?

Reply:
Only on those that are not stable IMO

Reply:
Opened a draft PR to start work on this here: https://github.com/open-telemetry/opentelemetry-collector/pull/11160